package/piksi_ins/overlay/usr/bin/start_pose_daemon

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+key_index=1
+
+b=/usr/bin
+l=/usr/lib
+
+caps="cap_ipc_lock,cap_sys_rawio+ep"
+
+nap_wrap --key-index $key_index \
+  --input-file $b/PoseDaemon.enc --output-file $b/PoseDaemon \
+  --permissions 0755 --capabilities "$caps" \
+  --input-file $b/local_pose_playback.enc --output-file $b/local_pose_playback \
```

Comment:
Should we be including "local_pose_playback" binary in the image at all?   If it is possible in this change I'd love to not install this binary since we don't need it in the product.

Reply:
I can delete it from the image, but it's installed by CRLs makefile, so we'd either need to change it there, or add a list of things to not install in ours

Reply:
Fixed